plenum/common/util.py

```diff
@@ -554,3 +554,24 @@ def min_3PC_key(keys) -> Tuple[int, int]:
 
 def max_3PC_key(keys) -> Tuple[int, int]:
     return max(keys, key=lambda k: (k[0], k[1]))
+
+
+from sortedcontainers import SortedDict as _SortedDict
+if 'peekitem' in dir(_SortedDict):
+    SortedDict = _SortedDict
+else:
+    # Since older versions of `SortedDict` lack `peekitem`
```

Comment:
Ok, it turned out that we use sortedcontainers==1.5.7 in the code, while we use canonical deb package for it, which is a version 1.4.4